lib/Text_Wiki/Text/Wiki/Parse/Default/Url.php

```diff
@@ -154,12 +164,25 @@ function parse()
         $postVars = $this->getConf("post_vars");
 
         if($postVars){
-        		// enable %%foo%% variables to act in described links
-        		$tmp_regex = 	'/\[(\*)?(%%[^%]+%%[^ ]*) ([^\]]+)\]/';
-        		$this->wiki->source = preg_replace_callback(
-	            $tmp_regex,
-	            array(&$this, 'processPV'),
-	            $this->wiki->source
+            // enable %%foo%% variables to act in described links
+            # What is this? I haven't seen it before
```

Comment:
For stuff like `%%title_linked%%` presumably?

Reply:
I'd expect to see that in listpages, not here. Besides, this requires a bracket and an optional asterisk. [It's not in the original Text_Wiki](https://github.com/pear/Text_Wiki/blob/master/Text/Wiki/Parse/Default/Url.php), so it must be wikidot-specific.